app/views/users/show.html.erb

```diff
@@ -1 +1,24 @@
-<h1>Here should be some more details about a specific user</h1>
\ No newline at end of file
+<div class="User">
+  <div class="info">
+    <img src=<%= @user.photo %> alt="user pic" />
+    <div class="title">
+      <h3><%= @user.name %></h3>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+  <div class="bio">
+    <h4>Bio:</h4>
+    <p><%= @user.bio %></p>
+  </div>
+  <div class="posts">
+    <% @user.last_3_posts.each_with_index do |post, i| %>
+      <div class="post">
+        <h4><%= link_to "Post #" + (i + 1).to_s, user_post_path(@user, post) %></h4>
+        <p><%= post.text %></p>
```

Comment:
- [ ] Kindly follow the given wireframe template for each user, this includes layout of components on the page as this is a requirement.

Given design
![image](https://user-images.githubusercontent.com/81932478/214705363-5e5c454f-9067-48b7-bf7e-de1cbc201ef6.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/214705633-323a7e2b-4bc1-411d-817c-aed0b8eff9af.png)